tests/material_model_dependencies_diffusion_dislocation/screen-output

```diff
@@ -2,44 +2,36 @@
 -- This is ASPECT, the Advanced Solver for Problems in Earth's ConvecTion.
 --     . version 1.4.pre
 --     . running in DEBUG mode
---     . running with 1 MPI process
+--     . running with 4 MPI processes
 --     . using Trilinos
 -----------------------------------------------------------------------------
 
-Loading shared library <./libmaterial_model_dependencies_diffusion_dislocation.so>
-
```

Comment:
you'll need to run this test via ctest, or it will not load the shared lib and not create the relevant output below (which also disappeared from your test output). Can you run it again and update the screen-output?